snippets/python/Diamond_Square.md

```diff
@@ -0,0 +1,49 @@
+# Diamond Square
+
+## Description
+A Program to print the Diamond Sqaure Pattern, number of rows is given as input.
+
+## Code
+```python
+# The Diamond Pattern
```

Comment:
This comment is not appropriate for the code you are referring to. As I can see you are taking the number of rows as input, you can simply quote it as 'rows input'

Reply:
Thanks , for the review , will upload the new file soon.